api/models.js

```diff
@@ -858,3 +858,20 @@ dlItemSchema.index({'dldataset': 1, 'dataset.datatype': 1, 'dataset.meta.subject
 exports.DLItems = mongoose.model('DLItems', dlItemSchema);
 
 
+var commentSchema = mongoose.Schema({
+    comment: String,
+    user_id: String, //sub of the user
+    create_date: {type: Date, default: Date.now},
```

Comment:
Please put dates toward the bottom of the schema (just to make it easier to find things)